src/controller/cve-id.controller/cve-id.controller.js

```diff
@@ -28,7 +28,7 @@ async function getCveId (req, res, next) {
         const cveId = JSON.parse(JSON.stringify(result)) // clone cve id returned from the db
         delete cveId._id
 
-        logger.info({ message: id + ' is ' + result.state + '. A 404 status was sent to the requester.', cveId: cveId })
+        logger.info({ uuid: req.ctx.uuid, message: id + ' is ' + result.state + '. A 404 status was sent to the requester.', cveId: cveId })
```

Comment:
I think we have to `JSON.stringify()` all of these structured log statements. Otherwise this is technically logging an anonymous JS object and not a string that is JSON. The difference is we could have logs that end up being [Object object] or lose the content on some of the values, depending on what they are. I may be wrong though.